README.md

```diff
@@ -2,23 +2,26 @@
 Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+1. Use `brew search` on the command line for a list of packages.
+2. Use `brew search --desc <keyword>` to browse packages from the command line.
+3. Visit [braumeister.org](http://braumeister.org) to browse packages online.
 
 ## More Documentation
-`brew help`, `man brew` or check [our documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
+Use `brew help` or `man brew` on the command line or visit [our documentation](https://github.com/Homebrew/brew/tree/master/share/doc/homebrew#readme).
 
 ## Troubleshooting
-First, please run `brew update` and `brew doctor`.
 
-Second, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Troubleshooting.md#troubleshooting).
+Before creating an issue, please run `brew update` (twice) and `brew doctor` (once) from the command line.
 
-**If you don't read these it will take us far longer to help you with your problem.**
+If the issue persists, read the [Troubleshooting Checklist](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Troubleshooting.md#troubleshooting).
+
+**If you don't read over these, it will take us far longer to help you with your problem.**
 
 ## Security
 Please report security issues to security@brew.sh.
 
+**If the issue is high risk, please use our PGP key.**
```

Comment:
Let's soften this a bit, something like `please consider using our...`. I'd rather not put someone off reporting something they think is suspicious because they'd need to go read PGP documentation first.